todo_app/app.py

```diff
@@ -2,47 +2,44 @@
 
 from todo_app.flask_config import Config
 
-from todo_app.data.trello_board import TrelloBoard
+from todo_app.data.todo_board import ToDoBoard
 
 import os
 
 def create_app():
     app = Flask(__name__)
     app.config.from_object('todo_app.flask_config.Config')
 
-    trello_board_id = os.getenv('TRELLO_BOARD_ID')
+    todo_board_id = os.getenv('TODO_BOARD_ID')
 
 
     @app.route('/')
     def index():
-        trello_board = TrelloBoard(trello_board_id)
-        items = trello_board.get_cards()
-        lists = trello_board.get_lists()
-        item_view_model = ViewModel(items,lists)
+        todo_board = ToDoBoard(todo_board_id)
+        items = todo_board.get_cards()
+        item_view_model = ViewModel(items)
         return render_template('index.html', view_model=item_view_model)
 
     
     @app.route('/items', methods=['POST'])
     def add_item():
         title = request.form.get('title')
-        list_id = request.form.get('idList')
-        trello_board = TrelloBoard(trello_board_id)
-        trello_board.create_card(title, list_id)
+        status = request.form.get('status')
+        todo_board = ToDoBoard(todo_board_id)
+        todo_board.create_card(title, status)
 
-        items = trello_board.get_cards()
-        lists = trello_board.get_lists()
-        item_view_model = ViewModel(items,lists)
+        items = todo_board.get_cards()
+        item_view_model = ViewModel(items)
         return render_template('index.html', view_model=item_view_model)
```

Comment:
Note how you had to change this here (and below) because your index function changed - if we adjusted this to redirect to the index then we could just rely on that functionality which means making these changes in just one place, as well as some other minor benefits (like refreshing the page not triggering a second POST)
```python
from flask import redirect
...

    return redirect('/')
```